tests/test_train.py

```diff
@@ -518,15 +572,13 @@ def test_model_finetuning_nlu_with_default_epochs(
     assert new_diet_metadata[EPOCHS] == DIETClassifier.defaults[EPOCHS] * 0.5
 
 
-@pytest.mark.parametrize("model_to_fine_tune", ["invalid-path-to-model", "."])
```

Comment:
mhm, can we still test the case that we have a directory without model files, please?

Reply:
I thought it was no longer valid because of removing the `try` but actually it was just breaking because I had an empty model in `.`. So I will add them back.